README.md

```diff
@@ -16,6 +16,18 @@ npm install --save-dev @datadog/datadog-ci
 yarn add --dev @datadog/datadog-ci
 ```
 
+If you need `datadog-ci` as a CLI tool instead of a package, you can run it with `npx` or install globally:
```

Comment:
Nit: you can add that `npx` will download it each time it is called (it seems to be the behavior on my computer but I am not 100% certain whether it is my config or default config).
Also you can add the link (https://www.npmjs.com/package/npx) to `npx`
```suggestion
If you need `datadog-ci` as a CLI tool instead of a package, you can run it with [`npx`](https://www.npmjs.com/package/npx) or install globally:
```